AssemblerLib/ComputeSparsityPattern.h

```diff
@@ -0,0 +1,40 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2015, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#ifndef ASSEMBLERLIB_COMPUTESPARSITYPATTERN_H
+#define ASSEMBLERLIB_COMPUTESPARSITYPATTERN_H
+
+#include <vector>
+
+#include "ProcessLib/NumericsConfig.h"
+
+namespace AssemblerLib
+{
+
+class LocalToGlobalIndexMap;
+
+using SparsityPattern = std::vector<GlobalIndexType>;
```

Comment:
Comment the purpose of the type something along the lines:
```c++
/// Vector of number of nonzero entries in global matrix.
```
and change the return description of `computeSparsityPattern()` below---now it is returning a `SparsityPattern`, not a vector.